### PR TITLE
fix: fall back to votes count if voters count is not supplied

### LINKED ISSUE #1125: Poll result percentage wrong
### Pre-Checks

- [X] Check that there isn't [already an issue](https://github.com/elk-zone/elk/issues) that reports the same bug to avoid creating a duplicate.
- [X] Check that this is a concrete bug. For Q&A open a [GitHub Discussion](https://github.com/elk-zone/elk/discussions) or join our [Discord Chat Server](https://chat.elk.zone).
- [X] Providing a screenshot or video to reproduce the issue or show visually what was meant.
- [ ] I am willing to provide a PR.

### Describe the bug

I looked at [this](https://front-end.social/@leaverou/109688267189424003) toot in [Elk](https://elk.zone/front-end.social/@leaverou/109688267189424003) and submitted the poll. This led to the results being shown. The percentage calculation on the graph was wrong though, including "2,400%", "0%", "400%", and "300%" (see screenshot).

### Reproduction video or screenshot

image

### Additional Context

Firefox Nightly, Windows 11

### PR DESCRIPTION
**Pull request by danielroe**
resolves #1125

### CODE REVIEW

File: components/status/StatusPoll.vue [hunk 1]
```diff
@@ -27,13 +27,18 @@ async function vote(e: Event) {
   }
   poll.voted = true
   poll.votesCount++
-  poll.votersCount = (poll.votersCount || 0) + 1
+
+  if (!poll.votersCount && poll.votesCount)
```
Comment (sxzz) [inline components/status/StatusPoll.vue:31]: `poll.votesCount` is always a number, not including `null`

Comment (danielroe) [reply components/status/StatusPoll.vue:31]: Is there a change you would suggest here?

Comment (antfu) [reply components/status/StatusPoll.vue:32]: Relavent to this? https://elk.zone/m.webtoo.ls/@cheeaun@mastodon.social/109688468147731159

Comment (danielroe) [reply components/status/StatusPoll.vue:32]: I think this PR is good to go. It doesn't matter if votersCount (or votesCount) is null or 0 - we fall back to zero in that case.

**Review by antfu — APPROVED** [review]

**Pull request merged**